README.md

```diff
@@ -34,46 +36,64 @@ Take a look at the file structure in this directory (Feel free to fork and clone
 
 ### `Gemfile`
 
-This holds a list of all the gems needed to run the application. The bundler gem provides us access to a terminal command `bundle install`. Bundler will look in the Gemfile and install any gems, as well as any gem dependencies for this application.
+This holds a list of all the gems needed to run the application. The bundler gem provides us access to a terminal command: `bundle install`. Bundler will look in the Gemfile and install any gems, as well as any gem dependencies for this application.
 
-### `config.ru` file
+Go ahead and enter this command in terminal. It will create a `Gemfile.lock` file for you, which locks in the current versions of each gem for your application. This way any updates of the gems in the future won't potentially break your app.
```

Comment:
So the Gemfile.lock doesn't do the version locking. It actually is just a documentation of what versions of the gem you have installed and should use. The lock word is actually because it makes sure that only one thing is running bundle install at a time!